code/modules/religion/sects/shadow_sect.dm

```diff
@@ -319,93 +325,80 @@
 	if(!rite_target)
 		return FALSE
 	rite_target.AddComponent(/datum/component/anti_magic, MAGIC_TRAIT, _magic = TRUE, _holy = FALSE)
-	//glowing wings overlay
-	to_chat(rite_target, "<span class='userdanger'>You are grateful to have been converted to the dark by [user]. Serve [user.real_name], and assist [user.p_them()] in completing [user.p_their()] goals at any cost.</span>")
+	if(rite_target != user)
+		to_chat(rite_target, "<span class='userdanger'>You are grateful to have been converted to the dark by [user]. Serve [user.real_name], and assist [user.p_them()] in completing [user.p_their()] goals at any cost.</span>")
+		var/objective = "You are grateful to have been converted to the dark by [user]. Serve [user.real_name], and assist [user.p_them()] in completing [user.p_their()] goals at any cost!"
+		brainwash(rite_target, objective, "Shadow Conversion")
+		log_game("[key_name(rite_target)] has been brainwashed with the objective '[objective]' via the chaplains sect shadow conversion.")
+	var/obj/effect/proc_holder/spell/targeted/shadowwalk/SW = new
```

Comment:
I reckon you should should put this after an early return, probably not too critical though

Reply:
Unsure what you mean here

Reply:
Essentially, after creating the shadowwalk thing, check if ``if(rite_target == user) return`` then do all the chat and brainwash stuff. Does that make sense?

Reply:
Figured it out